src/localServerApi.js

```diff
@@ -30,11 +30,34 @@ const createLocalServerApi = testkit => {
       res.sendStatus(200)
     })
     app.post(`/api/${project}/envelope/`, (req, res) => {
-      const { type, payload } = parseEnvelopeRequest(req.body)
-      if (type === 'transaction') {
-        testkit.transactions().push(transformTransaction(payload))
+      if (req.headers['transfer-encoding'] === 'chunked') {
```

Comment:
@ohana54 would you like to take a look as well?
I know you have extensive use of this